include/envoy/server/config_tracker.h

```diff
@@ -0,0 +1,69 @@
+#pragma once
+
+#include <functional>
+#include <map>
+#include <memory>
+
+#include "envoy/common/pure.h"
+
+#include "common/common/non_copyable.h"
+#include "common/protobuf/protobuf.h"
+
+namespace Envoy {
+namespace Server {
+
+/*
```

Comment:
nit: `/**` (same elsewhere if applicable). We should probably just check this with the linter at some point.

Reply:
Yeah, oops. Don't we just use vanilla clang-format? Or would we have control over this somewhere.

Reply:
/ worth filing an issue? By my count there are roughly 15 occurrences in trunk atm

Reply:
Sure, it's not a big deal, just noticed it.